aws-geo-location/src/main/java/com/amplifyframework/geo/location/options/AmazonLocationSearchByCoordinatesOptions.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.geo.location.options
+
+import com.amplifyframework.geo.options.GeoSearchByCoordinatesOptions
+
+class AmazonLocationSearchByCoordinatesOptions private constructor(
```

Comment:
nit: this could be a simple data class?

Reply:
unfortunately not due to the builder pattern used by its parent class (which is implemented in Java). we want to keep the instantiation pattern to be consistent across all the options in the library.